labs/byzantine-api-gateway.md

```diff
@@ -0,0 +1,26 @@
+# Lab Name
+_byzantine-api-gateway_
```

Comment:
you can change to byzantine api gateway

Reply:
Also remove all leading and trailing '_' in description, name and scope. As @trbs and checklist suggests please put the short description as the comment on the PR

Reply:
Yes, wil do